docs/book/src/plugins/go-v4-plugin.md

```diff
@@ -1,7 +1,7 @@
-# go/v4-alpha (go.kubebuilder.io/v4-alpha)
+# go/v4 (go.kubebuilder.io/v4)
 
-Kubebuilder will scaffold using the `go/v4-alpha` plugin only if specified when initializing the project. 
-This plugin is a composition of the plugins ` kustomize.common.kubebuilder.io/v2-alpha` and `base.go.kubebuilder.io/v4-alpha`. 
+Kubebuilder will scaffold using the `go/v4` plugin only if specified when initializing the project. 
+This plugin is a composition of the plugins ` kustomize.common.kubebuilder.io/v2` and `base.go.kubebuilder.io/v4`. 
```

Comment:
This PR seems to be focused on the go/v4-alpha --> go/v4 change. I would probably keep the kustomize/v2 reference as kustomize/v2-alpha (especially since there is [this PR](https://github.com/kubernetes-sigs/kubebuilder/pull/3234)):
```suggestion
This plugin is a composition of the plugins ` kustomize.common.kubebuilder.io/v2-alpha` and `base.go.kubebuilder.io/v4`. 
```